microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/internal/configordinal/properties/ConfigOrdinalPropertyProvider.java

```diff
@@ -0,0 +1,36 @@
+/*******************************************************************************
+* Copyright (c) 2022 Red Hat Inc. and others.
+*
+* This program and the accompanying materials are made available under the
+* terms of the Eclipse Public License v. 2.0 which is available at
+* http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+* which is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package org.eclipse.lsp4mp.jdt.internal.configordinal.properties;
+
+import org.eclipse.core.runtime.IProgressMonitor;
+import org.eclipse.lsp4mp.jdt.core.AbstractStaticPropertiesProvider;
+import org.eclipse.lsp4mp.jdt.core.MicroProfileCorePlugin;
+import org.eclipse.lsp4mp.jdt.core.SearchContext;
+
+/**
+ * Config ordinal property provider.
+ * 
+ * @see https://download.eclipse.org/microprofile/microprofile-config-1.4/microprofile-config-spec.html#_manually_defining_the_ordinal_of_a_built_in_configsource
+ */
+public class ConfigOrdinalPropertyProvider extends AbstractStaticPropertiesProvider {

```

Comment:
I will rename this class to `MicroProfileConfigPropertyProvider`  to embed future properties for the microprofile-config module.

Reply:
We have a class with the same name here: https://github.com/eclipse/lsp4mp/blob/51d5e8715c27a05c439fd84fe10e9cf3fd7b8bef/microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/internal/config/properties/MicroProfileConfigPropertyProvider.java#L52 

But it doesn't extend `AbstractStaticPropertiesProvider` which works with static properties. Would `MicroProfileConfigStaticPropertyProvider` make sense? And I could move this class in `eclipse/lsp4mp/jdt/internal/config/properties`